api/drivers/dummy/driver.js

```diff
@@ -245,6 +270,9 @@ class DummyDriver extends BaseDriver {
     return new Promise((resolve, reject) => {
       if (machine.status === 'error') {
         reject(machine.status);
+      } else if (machine.status === 'stopping') {
```

Comment:
The AWS sets a `stopping` status before setting a `stopped` status. How is this handled here ?